src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -1564,12 +1573,20 @@ public void DisableSearchTextBox()
         /// <summary>
         /// Clear after closing down
         /// </summary>
-        internal void Close()
+        internal void PackageManagerViewClose()
```

Comment:
Where is this function called?

Reply:
This is called from inside the `PackageManaverView`, when the view is closed down. Since the VM remains, but the view is closed, we are resetting some of the initial values.